app/models/solr_document.rb

```diff
@@ -67,24 +67,24 @@ def title
 
   # Returns the list of author names (ordered if possible)
   def authors
-    authors_ordered.map { |author| author["value"] }
+    authors_ordered.map(&:value)
   end
 
-  # Returns the list of authors with all their information
-  # including name and ORCID. (ordered if possible)
+  # Returns the list of authors with all their information including
+  # name, ORCID, and affiliation. List is ordered if possible.
   def authors_ordered
     @authors_ordered ||= begin
       authors_json = fetch('authors_json_ss', nil)
       if authors_json
         # PDC Describe records contain this field;
-        # det the author data and sort it.
-        authors = JSON.parse(authors_json)
-        authors.sort_by { |creator| creator["sequence"] }
+        # get the author data and sort it.
+        authors = JSON.parse(authors_json).map { |hash| Author.new(hash) }
```

Comment:
This is the "big" change since it switches the author information from a generic Ruby Hash to a specific `Author` class so that we can have more built-in logic to handle author information.